ming_sim/action_clusters.py

```
@@ -138,6 +141,33 @@ def cluster_by_kind(kind: str) -> Optional[ActionCluster]:
+
+def season_option_contract_prompt(kind: str) -> str:
+    """Human-facing season option contract projected from FieldSpec."""
+    cluster = cluster_by_kind(kind)
+    if cluster is None:
+        return ""
+    fields = [f for f in cluster.fields if f.season_option]
+    details = []
+    for spec in fields:
+        detail = spec.name
+        if spec.quantity_unit:
+            detail += f"（{spec.quantity_unit}）"
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)